aemcmc/negative_binomial_horseshoe_gibbs.py

```diff
@@ -43,7 +84,7 @@ def horseshoe_step(srng, beta, sigma2, lambda2_inv, tau2_inv):
         0.5 * (beta.shape[0] + 1),
         zeta_inv + 0.5 * (beta2 * lambda2_inv_new).sum() / sigma2,
     )
-    return lambda2_inv_new, tau2_inv_new, upsilon_inv, zeta_inv
```

Comment:
Don't we want to keep these last 2 parameters? I am thinking that this horseshoe step will be reused for implementing other types of regression models and thus some may make use of the values when updating other parameters in a single gibbs step.